src/WebJobs.Script.WebHost/DependencyInjection/JobHostServiceProvider.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using DryIoc;
+using DryIoc.Microsoft.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.DependencyInjection
+{
+    public class JobHostServiceProvider : IServiceProvider, IServiceScopeFactory, ISupportRequiredService, IDisposable
```

Comment:
Looks like all of this DI stuff belongs in WebJobs.Script

Reply:
Not this. Script does not have the scoped container we use in WebHost. So nothing defined here is needed there.